cluster/install/fluo-cluster/bin/impl/test.sh

```diff
@@ -37,6 +37,8 @@ git clone -b $TEST_BRANCH $TEST_REPO $FLUO_APP_NAME
 
 cd $FLUO_APP_NAME
 
+$FLUO_HOME/bin/fluo new $FLUO_APP_NAME
```

Comment:
Fixed in 43f695e. I made that commit w/ message `WIP` by accident.   I am finished making updates.